### PR TITLE
Updated Fall 2021 Information Join and Cadre

### PR DESCRIPTION
**Pull request by TJCaldwell**
Changed Spring 2021 to Fall 2021
Removed TSgt. Cauley

### CODE REVIEW

File: content/join.hbs [hunk 1]
```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Spring 2021</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Fall 2021</span> semester</h1>
 			<div class="columns">
 				<div class="column is-half side-by-side current-semester">
 					<h2>Join as a <span class="semester-text">Georgia Tech</span> student</h1>
-					<p>To enroll for Spring 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
+					<p>To enroll for Fall 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
```
Comment (petschekr) [inline content/join.hbs]: Do we have a new schedule document for Fall 2021?

File: content/join.hbs [hunk 2]
```diff
@@ -25,17 +25,17 @@
...
 					<a class="button is-large" href="files/Spring 2021 NSE GT Application.pdf">NSE Packet for Georgia Tech Students</a>
 				</div>
 				<div class="column is-half side-by-side future-semester">
 					<h2>Join as a <span class="semester-text">crosstown</span> student</h1>
 					<p class="college-list">Detachment 165 hosts students from:<br />Agnes Scott College, Clark Atlanta University, Clayton State University, Emory University, Georgia State University, Kennesaw State University, Morehouse College, Oglethorpe University, Spelman College, Mercer University, and University of West Georgia</p>
-					<p>To enroll for Spring 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
+					<p>To enroll for Fall 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
```
Comment (petschekr) [inline content/join.hbs]: Same for the enrollment packet

File: content/join.hbs [hunk 3]
```diff
@@ -59,32 +59,32 @@
...
+						<div class="day">19</div>
 					</div>
 					<p>NSE Package Due</p>
 				</div>
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">January</div>
-						<div class="day">14</div>
+						<div class="month">July</div>
+						<div class="day">5</div>
```
Comment (petschekr) [inline content/join.hbs]: Just want to confirm that In-Processing is July 5th which is before either of the other enrollment deadlines?

File: content/join.hbs [hunk 4]
```diff
@@ -59,32 +59,32 @@
...
 						AFROTC Day 1 &mdash; In-processing
 						<br />
-						<small>(0900&ndash;1500)</small>
+						<small>(0730&ndash;1500)</small>
 					</p>
 				</div>
 			</div>
 
-			<h1>Join in <span class="semester-text">Spring 2021</span> or later</h1>
+			<h1>Join in <span class="semester-text">Fall 2021</span> or later</h1>
```
Comment (petschekr) [inline content/join.hbs]: I think we talked about this last time but doesn't Spring 2022 make more sense since this link is for future interest?

**Review by petschekr — CHANGES_REQUESTED** [review]

File: content/join.hbs (hunk 3 above)
Comment (TJCaldwell) [reply content/join.hbs]: That was a confirmed error, I thought I fixed it to Aug 5, thank you!

File: content/join.hbs (hunk 2 above)
Comment (TJCaldwell) [reply content/join.hbs]: I asked, but there has not been any response yet. These were the only changes requested to be changed. I will check again

File: content/join.hbs (hunk 1 above)
Comment (TJCaldwell) [reply content/join.hbs]: Same as below. I will check again

File: content/join.hbs (hunk 4 above)
Comment (TJCaldwell) [reply content/join.hbs]: I see what you mean, I will inform Cadre. Thank you

**Review by petschekr — APPROVED** [review]
Looks good, thanks!

**Pull request merged**